openfeature-provider-local/src/main/java/com/spotify/confidence/flags/resolver/SidecarResolverServiceFactory.java

```diff
@@ -0,0 +1,97 @@
+package com.spotify.confidence.flags.resolver;
+
+import com.google.protobuf.Struct;
+import com.spotify.confidence.UnauthenticatedException;
+import com.spotify.confidence.flags.resolver.domain.AccountClient;
+import com.spotify.confidence.flags.resolver.domain.AccountState;
+import com.spotify.confidence.flags.resolver.domain.FlagToApply;
+import com.spotify.confidence.flags.resolver.domain.ResolvedValue;
+import com.spotify.confidence.flags.resolver.domain.ResolverState;
+import com.spotify.confidence.shaded.flags.resolver.v1.Sdk;
+import com.spotify.confidence.shaded.iam.v1.ClientCredential;
+import java.time.Instant;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicReference;
+import java.util.function.Supplier;
+import org.apache.commons.lang3.RandomStringUtils;
+
+public class SidecarResolverServiceFactory implements ResolverServiceFactory {
```

Comment:
Why would we have the SidecarResolverServiceFactory here?

Reply:
It should probably just be renamed